roles/system-base/files/opt/ivy/bash_functions.sh

```diff
@@ -113,6 +113,16 @@ bind_host: 0.0.0.0
 EOF
 }
 
+function enable_datadog_cloud_secutiry() {
```

Comment:
```suggestion
function enable_datadog_cloud_security() {
```

Simple spelling mistake